client/web/compose/src/views/Admin/Charts/Edit.vue

```diff
@@ -651,6 +654,24 @@ export default {
       }).catch(this.toastErrorHandler(this.$t('notification:chart.deleteFailed')))
     },
 
+    handleClone () {
+      const chart = this.chart.clone()
+      chart.chartID = NoID
+      chart.name = `${this.chart.name} (copy)`
+      chart.handle = `${this.chart.handle}_copy`
+
+      const resourceTranslationLanguage = this.currentLanguage
+
+      const c = Object.assign({}, chart, resourceTranslationLanguage)
+
+      this.createChart(c).then(({ chartID }) => {
+        this.toastSuccess(this.$t('notification:chart.saved'))
```

Comment:
Notification should be handled similarly to page layout cloning
`<resource> cloned and selected`

Same for fail
`Failed to clone <resource>